File: CodeChef/LongChallenge/AprilLongChallenge2020B/CARSELL.cpp
//https://www.codechef.com/APRIL20B/problems/CARSELL
//Accepted
#include <bits/stdc++.h>
using namespace std;

int main()
{

    int t;

    cin >> t;

    while (t--)
    {

        int n;
        long long int profit = 0;
        cin >> n;

        vector<int> v(n);

        for (int i = 0; i < n; i++)
        {
            cin >> v[i];
        }

        sort(v.rbegin(), v.rend());

        for (int i = 0; i < n; i++)
        {
            profit += max(v[i] - i, 0);
            profit %= 1000000007;
        }
        cout << profit << endl;
    }
    return 0;
}

File: CodeChef/LongChallenge/AprilLongChallenge2020B/CARSELL.js
//https://www.codechef.com/APRIL20B/problems/COVIDLQ
process.stdin.resume();
process.stdin.setEncoding('utf-8');
let arr = "";
process.stdin.on('data', function (chunk) {
    arr += chunk;
});
process.stdin.on('end', function () {

    arr = arr.split("\n");

    let testCases = arr[0];

    for (let i = 1; i <= 2 * testCases; i += 2) {
        let n = arr[i];
        let price = arr[i + 1].split(" ");

        price.sort(function (a, b) {
            return b > a;
        });
        console.log(price);


    }


});